HelloWorld.egg-info/dependency_links.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
Should these .txt files be part of the PR? This one for example looks like an empty file.